index.html

```diff
@@ -342,6 +353,32 @@ <h3 class="opacity-75 inline">wyniki wyszukiwania</h3>
           },
           showOptions () {
             this.options = !this.options
+          },
+          saveCacheToFile() {
+
+            const cache = localStorage.getItem('cached')
+            const fileName = 'powiaty.json'
+
+            const element = document.createElement('a');
```

Comment:
I'm not using semicolons in this project. I know that I should provide some linter for that, but I didn't know that anyone would like to contribute to this project. ;)